---
layout: seminar
title: "Developing a novel enzyme-based method to assess predictive methylation markers associated with severe SARS-CoV-2 pneumonia mortality"
speaker: "Iris Liu"
position: "Graduate Student"
institution: "Northwestern University"
department: "Pulmonary and Critical Care"
image: /assets/images/iris-liu.jpeg
keywords: "Genomics, Methyl-seq, T cell"
authors: "Iris Liu<sup>1,2</sup>, Kathryn A. Helmin<sup>1</sup>, Zachary D. Dortzbach<sup>1</sup>, Carla P. Reyes Flores<sup>1,2</sup>, Manuel A. Torres Acosta<sup>1,3</sup>, Jonathan K. Gurkan<sup>1,2,3</sup>, Anthony M. Joudi<sup>1</sup>, Nurbek Mambetsariev<sup>4</sup>, Luisa Morales-Nebreda<sup>1</sup>, Mengjia Kang<sup>1</sup>, Luke Rasmussen<sup>5</sup>, Xóchitl G. Pérez-Leonor<sup>1</sup>, Hiam Abdala-Valencia<sup>1</sup>, and Benjamin D. Singer<sup>1,6,7,8</sup>"
affiliations: "1. Division of Pulmonary and Critical Care Medicine, Northwestern University Feinberg School of Medicine; 2. Driskill Graduate Program, Northwestern University Feinberg School of Medicine; 3. Medical Scientist Training Program, Northwestern University Feinberg School of Medicine; 4. Division of Allergy and Immunology, Northwestern University Feinberg School of Medicine; 5. Division of Health and Biomedical Informatics, Department of Preventive Medicine, Northwestern University Feinberg School of Medicine; 6. Department of Biochemistry and Molecular Genetics, Northwestern University Feinberg School of Medicine; 7. Simpson Querrey Institute for Epigenetics, Northwestern University Feinberg School of Medicine; 8. Simpson Querrey Lung Institute for Translational Science (SQ LIFTS), Northwestern University Feinberg School of Medicine"
abstract: "Commonly used bisulfite-based procedures for DNA methylation sequencing can degrade DNA, worsening signal-to-noise ratios in samples with low DNA input. Enzymatic methylation sequencing (EM-seq) has been proposed as a less biased alternative for methylation profiling with greater genome coverage. Reduced representation approaches enrich samples for CpG-rich genomic regions, thereby enhancing throughput and cost effectiveness. We hypothesized that enzyme-based technology could be adapted for reduced representation methylation sequencing to enable DNA methylation profiling of low-input samples. We leveraged the well-established differences in methylation profile between mouse CD4+ T cell populations to compare the performance of our reduced representation EM-seq (RREM-seq) procedure against an established reduced representation bisulfite sequencing (RRBS) protocol. While the RRBS method failed to generate reliable DNA libraries when using <2 ng of DNA, the RREM-seq method successfully generated reliable DNA libraries from 1–25 ng of mouse and human DNA. Low-input (≤2-ng) RREM-seq libraries demonstrated superior regulatory genomic element coverage compared with RRBS libraries with more than 10-fold higher DNA input. RREM-seq also successfully detected lineage-defining methylation differences between alveolar conventional T and regulatory T cells obtained from patients with severe SARS-CoV-2 pneumonia and could be applied to identify molecular signatures associated with severe SARS-CoV-2 pneumonia outcome. Our RREM-seq method enables single-nucleotide resolution methylation profiling using low-input samples, including from clinical sources."
date: 2026-04-10
zoom_link:
calendar_link: /assets/cal/2026-04-10.ics
recording_link:
---
*Bio coming soon.*
